docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
     image: postgres:${POSTGRES_VERSION}
     restart: ${RESTART}
     volumes:
+      - postgresql-data:/var/lib/postgresql/data
```

Comment:
thats not needed as the backup works over a network connection

Reply:
Same as redis volume, but in fact adding this volume breaks backup (